docs/tool-prerequisites.md

```diff
@@ -47,3 +47,10 @@ mpdev /scripts/doctor.py
 ```
 
 Save your gcloud project to an environment variable.
+
+
+## Install Application CRD
```

Comment:
can you move this to before the doctor tool?

Reply:
Done!